### PR TITLE
Don't clean items on enter if autoCloseOnEnter is disabled

### PR DESCRIPTION
**Pull request by jirkalhotka**
If autoCloseOnEnter is disabled, the picker stays open with user text typed in when user presses enter, so it makes sense to leave their selection.

### CODE REVIEW

**Review by 1egoman — APPROVED** [review]
Code looks fine, I'd like to give it a try this weekend before merging though!

**Comment by jirkalhotka** [comment]
Thanks for the merge! I saw you bumped the version number up to 1.2.0, when are you planning to release?:)

**Comment by 1egoman** [comment]
Oh oops, I thought I did! Let me do that now!

**Comment by 1egoman** [comment]
Published `react-fuzzy-picker@1.2.0`!

**Comment by jirkalhotka** [comment]
Amazing! Wish all maintainers were as responsive as you are! Keep up the great work!

**Pull request merged**
